controllers/auth_config_status_updater.go

```diff
@@ -223,3 +220,27 @@ func issuingWristbands(authConfig *api.AuthConfig) bool {
 	}
 	return false
 }
+
+func capitalize(message string) string {
```

Comment:
These generic methods, better to be publicly available in `commons` or `utils` package.

Reply:
I'm really afraid of having these kinds of packages and be opening the door for a whole miscellaneous of functions without any logical grouping. Why wouldn't `github.com/kuadrant/authorino/pkg/json` be in there, right?

Maybe introduce `github.com/kuadrant/authorino/pkg/strings` and `github.com/kuadrant/authorino/pkg/slices` or that would be overkill?

Reply:
Okay. I ended up moving to `github.com/kuadrant/authorino/pkg/utils`, but I have to say. I don't have a good feeling about this. 😕

Reply:
you will not regret :)